CHANGES.md

```diff
@@ -10,6 +10,8 @@
 
 ### Removed
 
+- Remove support for delegates (#428, @NathanReb)
```

Comment:
Maybe we can mention `delegate-info` here just to make sure that folks who were using delegates know how to adapt to the new situation? Something like "(...) You can use the `delegate-info` workflow instead."

Reply:
Very good point, I'll add that! I'm also wondering if we shouldn't have named this command differently as delegate might not be meaningful for most users but that's a separate matter! We can still add another alias and remove `delegate-info` in its favor later on!